ghcup-vanilla-0.0.8.yaml

```diff
@@ -265,7 +265,7 @@ toolRequirements:
           - gmake
           - ncurses
           - perl5
-          - pkgconf
+          - pkg-config
```

Comment:
Why?

Reply:
Apologies, this is very likely a rebasing error.